docs/experimentalists/sampler/random/index.md

```diff
@@ -0,0 +1,10 @@
+# Random Sampler
```

Comment:
No example (apart from example code) because this one is so self-explanatory?

Reply:
yes  thought it was to obvious for an example